conf/locale/locale_de-DE.ini

```diff
@@ -915,9 +915,9 @@ dashboard.current_goroutine=Aktuelle Goroutines
 dashboard.current_memory_usage=Aktuelle Speichernutzung
 dashboard.total_memory_allocated=Zugeteilter Gesamtspeicher
 dashboard.memory_obtained=Erhaltener Speicher
-dashboard.pointer_lookup_times=Pointer Lookup Times
-dashboard.memory_allocate_times=Memory Allocate Times
-dashboard.memory_free_times=Memory Free Times
+dashboard.pointer_lookup_times=Zeigerlookup-Dauern
```

Comment:
It's a key for a table in the admin ui where you can see the garbage collection interval and so on. So it should be Dauer